kong/conf_loader/constants.lua

```diff
@@ -513,7 +513,7 @@ local CONF_PARSERS = {
   cluster_use_proxy = { typ = "boolean" },
   cluster_dp_labels = { typ = "array" },
   cluster_rpc = { typ = "boolean" },
-  cluster_incremental_sync = { typ = "boolean" },
```

Comment:
Should we also include this change in doc?

Reply:
It is an undocumented flag currently.